.github/workflows/package.yml

```diff
@@ -262,6 +267,9 @@ jobs:
       - build
       - test-elastic
       - test-opensearch
+    env:
+      IS_DEPLOYABLE: ${{ secrets.AR_DOCKER_USERNAME != '' }}
+    if: needs.build.outputs.code_changes == 'true' && env.IS_DEPLOYABLE == 'true'
```

Comment:
If it works, then we can also remove the `env` from here: if `IS_DEPLOYABLE` would ever be false, then `build` would have been skipped, and the `docker_deploy` job would never have started as `build` is one of its requirements.